ibm/service/eventnotification/resource_ibm_en_smtp_user.go

```diff
@@ -45,6 +45,11 @@ func ResourceIBMEnSMTPUser() *schema.Resource {
 				Optional:    true,
 				Description: "SMTP confg Id.",
 			},
+			"username_to_clone": &schema.Schema{
+				Type:        schema.TypeString,
```

Comment:
It's optional and there is no update change, what happens in case of update ?

Reply:
Added ForceNew for parameter username_to_clone ,
it will destroy the previous user and clone new one with updated username.